dipy/workflows/denoise.py

```diff
@@ -55,6 +55,8 @@ def run(self, input_files, bval_files, model='ols',
             Regularization parameter only for ridge regression model.
         verbose : bool, optional
             Show progress of Patch2Self and time taken.
+        patch_radius : int or 1 D array, optional
```

Comment:
`variable int, optional` instead 

you can look at this example: https://github.com/dipy/dipy/blob/976217e547c4da6bfb210642adbb071c2ee40dd9/dipy/workflows/denoise.py#L277